doc/nrf/security/trusted_storage.rst

```diff
@@ -7,9 +7,14 @@ Trusted storage in the |NCS|
    :local:
    :depth: 2
 
-There are several options for storing keys and other important data persistently when developing applications with the |NCS|.
-Different storage options have different features.
-One of the options is to use the :ref:`trusted_storage_readme` library.
+:ref:trusted_storage_in_ncs is a security mechanism designed to securely store and manage sensitive data, such as cryptographic keys, device credentials, and configuration data.
+The mechanism is essential for IoT devices, as it allows the implementation of secure communication between devices.
+
+The Trusted Storage can utilize one of the following backends to implement PSA Certified Secure Storage API:
+
+* TF-M Platform Root of Trust (PRoT).
+  This can only be utilized if the `ARM TrustZone`_ technology and hardware-accelerated firmware isolation are supported by the platform in use.
```

Comment:
```suggestion
  This can only be used if the `ARM TrustZone`_ technology and hardware-accelerated firmware isolation are supported by the platform in use.
```